tests/wpt/metadata/html/dom/interfaces.html.ini

```diff
@@ -10536,3 +10284,255 @@
+    expected: FAIL
+
+  [Document interface: calling adoptNode(Node) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling createAttribute(DOMString) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling createAttributeNS(DOMString,DOMString) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling createEvent(DOMString) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling createNodeIterator(Node,unsigned long,NodeFilter) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling createTreeWalker(Node,unsigned long,NodeFilter) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling getElementsByName(DOMString) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling getElementById(DOMString) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling prepend([object Object\],[object Object\]) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling append([object Object\],[object Object\]) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling querySelector(DOMString) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Document interface: calling querySelectorAll(DOMString) on iframe.contentDocument with too few arguments must throw TypeError]
+    expected: FAIL
+
+  [Node interface: iframe.contentDocument must inherit property "ELEMENT_NODE" with the proper type (0)]
+    expected: FAIL
+
+  [Node interface: iframe.contentDocument must inherit property "ATTRIBUTE_NODE" with the proper type (1)]
+    expected: FAIL
+
+  [Node interface: iframe.contentDocument must inherit property "TEXT_NODE" with the proper type (2)]
+    expected: FAIL
+
+  [Node interface: iframe.contentDocument must inherit property "CDATA_SECTION_NODE" with the proper type (3)]
+    expected: FAIL
+
+  [Node interface: iframe.contentDocument must inherit property "ENTITY_REFERENCE_NODE" with the proper type (4)]
+    expected: FAIL
+
+  [Node interface: iframe.contentDocument must inherit property "ENTITY_NODE" with the proper type (5)]
+    expected: FAIL
+
+  [Node interface: iframe.contentDocument must inherit property "PROCESSING_INSTRUCTION_NODE" with the proper type (6)]
+    expected: FAIL
+
+  [Node interface: iframe.contentDocument must inherit property "COMMENT_NODE" with the proper type (7)]
```

Comment:
Does this mean that `iframe.contentDocument` on an about:blank iframe is still returning null because it thinks it's cross-origin?